CONTRIBUTING.en.md

```diff
@@ -0,0 +1,418 @@
+# Contributor's Guide
+
+[Japannese](./CONTRIBUTING.md)
+
+VOICEVOX ENGINE is an open-source project. This project is actively developed, and its results are reflected in the production version of VOICEVOX. VOICEVOX ENGINE welcomes contributions from the community.  
+This guide provides information to assist contributors, including development policies, pull request procedures, and review processes.
+
+## Table of Contents
+
+You can check the guidelines for VOICEVOX ENGINE policies from the following:
+
+- [Development Governance](#development-governance)
+- [Versioning](#versioning)
+- [Branch Strategy](#branch-strategy)
+- [Pull Requests](#pull-requests)
+- [Reviews](#reviews)
+- [Bugs](#bugs)
+- [Feature Improvements](#feature-improvements)
+- [Static Analysis](#static-analysis)
+- [Tests](#tests)
+- [License](#license)
+
+You can check the guidelines according to the contributor's purpose from the following:
+
+- [Submitting a Pull Request](#submitting-a-pull-request)
+- Bugs
+  - [Finding Bugs](#finding-bugs)
+  - [Reporting Bugs](#reporting-bugs)
+  - [Fixing Bugs](#fixing-bugs)
+- Feature Improvements
+  - [Finding Feature Improvement Tasks](#finding-feature-improvement-tasks)
+  - [Requesting Features](#requesting-features)
+  - [Implementing Feature Improvements](#implementing-feature-improvements)
+- [Setting Up the Environment](#setting-up-the-environment)
+- [Running the Code](#running-the-code)
+
+You can check frequently used commands for development from the following:
+
+- [Installing Dependencies](#installing-dependencies)
+- [Running Without Voice Library](#running-without-voice-library)
+- Packages
+  - [Adding Packages](#adding-packages)
+  - [Updating Packages](#updating-packages)
+  - [Reflecting Package Information to pip requirements.txt File](#reflecting-package-information-to-pip-requirementstxt-file)
+- Static Analysis
+  - [Checking for Typos](#checking-for-typos)
+  - [Running Static Analysis in Batch](#running-static-analysis-in-batch)
+- Tests
+  - [Testing the Code](#testing-the-code)
```

Comment:
```suggestion
  - [Testing Code](#testing-code)
```
This item seems not to match with the heading on actual body (also the anchor was not working).

Reply:
Ah, thank you.